src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -48,6 +48,8 @@ public class DrivetrainSubsystem extends SubsystemBase {
   // private final AHRS navx = new AHRS(SPI.Port.kMXP, (byte) 200); // NavX connected over MXP
   private final WrappedGyro gyro = new WrappedGyro(GyroType.PIGEON);
```

Comment:
Consider adding GyroType to the DriveTrainConstants, with the value that is correct for Hana / springtrap / calliope / new, and create the gyro in the DriveTrain constructor